packages/terra-theme-provider/tests/nightwatch/GlobalSwitchThemes.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import ThemeProvider from '../../lib/ThemeProvider';
+import MockThemeComponent from './MockThemeComponent';
+
+
+class SwitchThemes extends React.Component {
```

Comment:
Should we enhance this component to track the current site theme upon render?  

If you got to the deployment and change the theme to MOCK, go to this test example and change the themes. When you return to the site home page, the theme there is also changed and is no longer the MOCk theme.

Reply:
^ This does show the global theme switching works 😄

Reply:
Talked offline and plan to keep as is.